website/docs/getting-started/installation.md

```diff
@@ -0,0 +1,201 @@
+---
+id: installation
+title: ChaosCenter Cluster Scope Installation
+sidebar_label: Installation
+---
+
+---
+
+## Prerequisites
+
+Before deploying LitmusChaos, make sure the following items are there
+
+- Kubernetes 1.15 or later
+
+- A Persistent volume of 20GB
+
+  >Recommend to have a Persistent volume(PV) of 20GB, You can start with 1GB for test purposes as well. This PV is used as persistent storage to store the chaos config and chaos-metrics in the Portal. By default, litmus install would use the default storage class to allocate the PV. Provide this value
+
+- [Helm3](https://v3.helm.sh/) or [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+
+## Installation
+
+Installation of Litmus can be done using either of the below methods
+
+- [Helm3](#install-litmus-using-helm) chart
+- [Kubectl](#install-litmus-using-kubectl) yaml spec file
+
+### Install Litmus using Helm
+
+The helm chart will install all the required service account configuration and ChaosCenter.
+
+The following steps will help you install Litmus ChaosCenter via helm.
+
+#### Step-1: Add the litmus helm repository
+
+```bash
+helm repo add litmuschaos https://litmuschaos.github.io/litmus-helm/
+helm repo list
+```
+
+#### Step-2: Create the namespace on which you want to install Litmus ChaosCenter
+
+- The litmus infra components will be placed in this namespace.
+
+> The ChaosCenter can be placed in any namespace
+
+```bash
+kubectl create ns <LITMUS_PORTAL_NAMESPACE>
```

Comment:
If Replace <LITMUS_PORTAL_NAMESPACE> with litmus, the user experience will  be good.

Reply:
Sure, have replaced this in Getting Started Installation for UX.

The approach for Advance Installation have been kept using <LITMUS_PORTAL_NAMESPACE> which this note
> If you are installing Litmus in any other namespace apart from litmus namespace, make sure to change the same in the manifest too https://litmuschaos.github.io/litmus/2.0.0-Beta/litmus-2.0.0-Beta.yaml.